tests/gen_cert_test.py

```diff
@@ -154,7 +154,7 @@ def test_cert_date_timezone():
     """
     Make sure certs render dates according to the passed timezone
     """
-    today_date = datetime.datetime(2016, 8, 22, 0, 0, 0, 0)
+    today_date = '2016-08-22'
     utc_cert_date = get_cert_date(today_date, timezone='UTC')
```

Comment:
that's not gonna fix it though, cause if I pass it ANY date string, it's NOT gonna apply the timezone, which is correct